packages/card/CtaCard.jsx

```diff
@@ -66,6 +66,7 @@ export default function CtaCard({
           >
             {eyebrow && <span className="su-sr-only">{`${eyebrow}:`}</span>}
             {title}
+            <span className="su-sr-only">(link is external)</span>
```

Comment:
I added screen reader only text "(link is external)" right after the link text for the CTA card. I think the placement of this is also better than before since this text is about the link itself, so it makes more sense that it is right after the link text.